AIPscan/Data/data.py

```diff
@@ -6,7 +6,6 @@
 
 VALID_FILE_TYPES = set(item.value for item in FileType)
 
-FIELD_AIP = "AIP"
```

Comment:
I think this is still in use in a couple places in the codebase that would need to be changed before we can safely remove it:
* https://github.com/artefactual-labs/AIPscan/blob/91a8237998e0edb4ce79a4419878f92a21fe9908/AIPscan/Reporter/report_largest_files.py#L29
* https://github.com/artefactual-labs/AIPscan/blob/91a8237998e0edb4ce79a4419878f92a21fe9908/AIPscan/Reporter/helpers.py#L31

I think those were both my doing, so mea culpa! Feel free to change those to `data.FIELD_AIP_NAME` and delete this, or just leave `FIELD_AIP` in for now and I'll clean it up, as you prefer.

Reply:
I also only counted those two so have changed them and tried the largest file report and it looks okay...  so :crossed_fingers: we got that.